generators/app/templates/terra-package.json

```diff
@@ -25,12 +25,15 @@
     "terra-toolkit": "^0.x"
   },
   "peerDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "prop-types": "^15.5.8",
```

Comment:
I think we should move `prop-types` to a dependency.

https://github.com/cloudflare/cf-ui/blob/master/packages/cf-component-box/package.json#L17